packages/@lwc/engine-dom/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "../../../tsconfig.json",
+
+    "compilerOptions": {
+        "outDir": "types",
+        "lib": ["dom", "esnext"]
```

Comment:
This is set to `es2018` in engine-core. Which should we standardize on? Seems safer to avoid `esnext` due to it being a moving target but not sure.

According to the documentation, [`files, include and exclude from the inheriting config file overwrite those from the base config file`](https://www.typescriptlang.org/docs/handbook/compiler-options.html) so it sounds like we should be able to move the ES version to the root `tsconfig.json` file and add the `dom` lib where necessary?

Reply:
Looked into this today and the value of `lib` from the base config file is indeed overwritten. Seems like we should be going with `es2018`: https://github.com/salesforce/lwc/pull/1918